one-to-one-scheduler/one-to-one-scheduler.gs

```diff
@@ -626,6 +633,133 @@ function sendInvitesToMentors() {
   showInfo(TITLE_SUCCESS, summary);
 }
 
+function getRecordingLinks() {
+  let calendar = getCalendar();
+  if (!calendar) {
+    showInfo(TITLE_ERROR, MSG_CALENDAR_NOT_FOUND);
+    return;
+  }
+
+  if (!confirmCalendar(calendar)) {
+    showInfo(TITLE_ERROR, MSG_CALENDAR_REJECTED);
+    return;
+  }
+
+  let activeSheet = SpreadsheetApp.getActiveSheet();
+  let oneToOneData = get1to1Data(activeSheet);
+  if (!oneToOneData) {
+    showInfo(TITLE_ERROR, MSG_COULD_NOT_FETCH_DATA);
+    return;
+  }
```

Comment:
This block of code appears in multiple places. Please refactor this into a separate function.
```js
function initSetup() {
  let result = {
    success: false,
    calendar: null,
    activeSheet: null,
    oneToOneData: null
  };
  result.calendar = getCalendar();
  if (!result.calendar) {
    showInfo(TITLE_ERROR, MSG_CALENDAR_NOT_FOUND);
    return result;
  }

  if (!confirmCalendar(result.calendar)) {
    showInfo(TITLE_ERROR, MSG_CALENDAR_REJECTED);
    return result;
  }

  result.activeSheet = SpreadsheetApp.getActiveSheet();
  result.oneToOneData = get1to1Data(result.activeSheet);
  if (!result.oneToOneData) {
    showInfo(TITLE_ERROR, MSG_COULD_NOT_FETCH_DATA);
    return result;
  }
  result.success = true;
  return result;
}
```

Reply:
`initSetup()` added